bin/python_modules/integronfinder_process.py

```diff
@@ -36,13 +36,17 @@ def integron_parser(mge_data, integron_results, inf_gbks):
                                         mge_counter += 1
                                         mge_id = "inf_" + str(mge_counter)
                                         start = int(feature.location.start)
+                                        if start == 0:
```

Comment:
No need to check `start == 0` you can just do `if not start`
for flag better use boolean True and False values.
I guess your assumption is that `feature.type == "integron"` usually comes before "attC" because you are assigning `mge_id` only there. If you meet `attC` before `integron` then `mge_id` can come even from previous `gbk_file`. But maybe I misunderstand a technique.
I have a gut feeling that those flags and checks can be simlified. But I need a file example to help you :)

Reply:
Hello! Thank you for taking a look. In that block of code, we are parsing a gbk file. Find attached an example of the summary file and a gbk of a contig with a complete integron.

[contig_5851.gbk.gz](https://github.com/EBI-Metagenomics/mobilome-annotation-pipeline/files/15185828/contig_5851.gbk.gz)
[contigs.summary.gz](https://github.com/EBI-Metagenomics/mobilome-annotation-pipeline/files/15185830/contigs.summary.gz)